adapter/test/stdio/stdio_adapter_mpi_test.cpp

```diff
@@ -156,10 +156,24 @@ int pretest() {
   if (fs::exists(info.existing_file)) fs::remove(info.existing_file);
   if (fs::exists(info.existing_file)) fs::remove(info.existing_file);
   if (fs::exists(info.existing_file_cmp)) fs::remove(info.existing_file_cmp);
-  if (!fs::exists(info.existing_file)) {
+  fs::path temp_fullpath = "/tmp";
+  temp_fullpath /= args.filename;
+  std::string temp_ext_file = temp_fullpath.string() + "_temp_" +
+                              std::to_string(info.rank) + "_of_" +
+                              std::to_string(info.comm_size);
```

Comment:
Should `temp_ext_file` have the PID appended?

Reply:
I have appended.